pkgs/development/web/bun/default.nix

```diff
@@ -64,4 +64,38 @@ stdenvNoCC.mkDerivation rec {
     maintainers = with maintainers; [ DAlperin jk ];
     platforms = builtins.attrNames dists;
   };
```

Comment:
By convention, `meta` is the last attribute in packages, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.